packages/arb-token-bridge-ui/src/util/__tests__/NumberUtils.test.ts

```diff
@@ -0,0 +1,94 @@
+import { BigNumber } from 'ethers'
+import { formatTokenBalance } from '../NumberUtils'
+
+describe('formatTokenBalance', () => {
+  describe('for short token symbol', () => {
+    const getResult = (value: string) =>
+      formatTokenBalance({
+        symbol: 'ETH',
+        balance: BigNumber.from(value),
+        decimals: 6
+      })
+
+    it('should return 0 for value = 0', () => {
+      expect(getResult('0')).toBe('0')
+    })
+
+    it('should return the full number with 5 decimals for value lower than 1', () => {
+      expect(getResult('123456')).toBe('0.12346')
+    })
```

Comment:
add a TODO case for a very small amount like `0.00003 ETH`?

Reply:
I added new test cases for really low balance